B827EBFFFE8A6621.json

```diff
@@ -1,6 +1,6 @@
 {
     "gateway_conf": {
-    "gateway_ID": "B827EBFFFE8A6621",
+    "gateway_ID": "TTN-KBS-Bensheim-03",
```

Comment:
This will not work, the ID must be the MAC-address-like identifier of the gateway.